src/lib.rs

```diff
@@ -5,7 +5,7 @@ use near_sdk::{near, Gas, NearToken};
 mod deploy;
 mod manager;
 
-const NEAR_PER_STORAGE: NearToken = NearToken::from_yoctonear(10u128.pow(18)); // 10e18yⓃ
+const NEAR_PER_STORAGE: NearToken = NearToken::from_yoctonear(10u128.pow(19)); // 10e18yⓃ
```

Comment:
without this i wasn't seeing a message like 
```bash
Error:
   0: Error: An error occurred during a `FunctionCall` Action, parameter is debug message.
      ExecutionError("Smart contract panicked: panicked at src/deploy.rs:41:9:\nAttach at least 1.55 NEAR ")
```

Reply:
I don't feel the error message and this line are not related,

Reply:
could you give an example with a broken test using workspaces? (i.e. a broken test on `tests/sandbox.rs` that way we could see the error and debug it easily

Reply:
@gagdiez done. added test in https://github.com/near-examples/factory-rust/pull/4/commits/d0ab63f1e9e7b7fca7d4496396403531978f9ea7 , which passes for both 18 and 19 values )))
happily, `sandbox` behaviour is consistent with `testnet` (not 146% sure about mainnet, but should be the same 1 NEAR per 100 Kb), 
but i've noticed `testnet` behaviour of this example to be confusing, when the value is 18